### ISSUE TITLE
Installation process resets/disables all USB devices when installing to an USB external device

### ISSUE THREAD

**Issue by msmafra** [calamares/calamares#2349]
**Describe the bug**
Installation process resets and/or disables all USB devices when installing to an USB external device (case with SSD). It seems that it disables USB plugged drives/devices when in the process of writing the new partition layout. It resets/stops USB on my system the keyboard (including the laptop's one), mouse and the USB case where the system would've been installed.

**To Reproduce**
1. plug a USB case with a SSD or HDD;
2. Start the installation;
3. Confirm to start it;

**Expected behavior**
To be able to install the system on a external drive/device (case, dock etc);

**Screenshots and Logs**

[cachyos-install.webm](https://github.com/CachyOS/cachyos-calamares/assets/1005457/c9e1cbde-e724-458e-ae6b-7a9b97ba8c95)
![IMG_20240627_091217-2](https://github.com/CachyOS/cachyos-calamares/assets/1005457/a8688f1c-2591-498b-967b-ec944089ede3)
![IMG_20240627_091208-2](https://github.com/CachyOS/cachyos-calamares/assets/1005457/f59fbebd-9a2f-4c8f-bff0-d5742bf4db40)

**Additional context**
Using the cachyos-kde-linux-240609.iso and also reported it there, but seems not related to their customizations.
https://termbin.com/vlvp

**Comment by ptr1337** [comment]
Just a little note, it appears that using xfs as filesystem did not show this issue.

**Comment by msmafra** [comment]
> Just a little note, it appears that using xfs as filesystem did not show this issue.

Yes, I forgot about that, both BTRFS and XFS worked without encryption.

**Comment by adriaandegroot** [comment]
It sounds to me like something -- well, formatting a disk and then creating a filesystem on it will do so -- is triggering a re-scan of USB devices, potentially followed by auto-mount or auto-scan of the USB device. You can test this manually, by (re)creating the partition table and creating a partiton and filesystem on the device. Then watch what happens from whatever you live-desktop environment is.
